src/components/Analysis/Applications/CreateApplicationDialog.vue

```diff
@@ -281,9 +341,23 @@ export default {
 
   computed: {
     ...mapState(["userToken", "config"]),
-    ...mapState("analysisModule", ["computeNodes"]),
+    ...mapState("analysisModule", ["computeNodes"])
+    // cpuItems() {
```

Comment:
please remove commented out code or write a comment explaining why it is commented out. For this one, I'll take care of it in the epic branch though!